www/include/reporting/dashboard/DB-Func.php

```diff
@@ -586,13 +585,20 @@ function getLogInDbForServicesGroup($servicegroupId, $startDate, $endDate, $repo
             $serviceGroupStats[$hostServiceid][$name] = 0;
         }
 
+        //Must be here to display link for redirection to a specific server if there is no link between host and server
         $serviceGroupStats[$hostServiceid]["HOST_ID"] = $hostId;
         $serviceGroupStats[$hostServiceid]["SERVICE_ID"] = $serviceId;
         $serviceGroupStats[$hostServiceid]["HOST_NAME"] = $service['host_name'];
         $serviceGroupStats[$hostServiceid]["SERVICE_DESC"] = $service['service_description'];
 
         if (isset($servicesStats[$hostId][$serviceId])) {
             $serviceGroupStats[$hostServiceid] = $servicesStats[$hostId][$serviceId];
+
+            //Must be here to display link for redirection to a specific server if there is a link between host and server
+            $serviceGroupStats[$hostServiceid]["HOST_ID"] = $hostId;
+            $serviceGroupStats[$hostServiceid]["SERVICE_ID"] = $serviceId;
+            $serviceGroupStats[$hostServiceid]["HOST_NAME"] = $service['host_name'];
+            $serviceGroupStats[$hostServiceid]["SERVICE_DESC"] = $service['service_description'];
```

Comment:
duplicated with lines 589-592

Reply:
No host and service will appear below the pie chart if I delete lines 498-601

Reply:
fixed